src/main.rs

```diff
@@ -65,10 +68,22 @@ fn init_pool() -> Option<DbPool> {
 }
 
 fn main() {
-
     let dbpool = init_pool().expect("main: database pool initialization error");
     let workpool = ScheduledThreadPool::with_name("worker {}", num_cpus::get());
-    let searcher = Arc::new(UnmanagedSearcher::open(&"search_index").unwrap());
+    let searcher = match UnmanagedSearcher::open(&"search_index") {
+        Err(Error::Search(e)) => match e {
+            SearcherError::WriteLockAcquisitionError => panic!(
+r#"Your search index is locked. Plume can't start. To fix this issue, run:
```

Comment:
I should have made this message from the start. Maybe add something about making sure no other instance of plume is already running here?